gooey/gui/components/widgets/core/chooser.py

```diff
@@ -58,37 +58,53 @@ def setValue(self, value):
     def getValue(self):
         return self.widget.getValue()
 
+    def getChooserOptions(self):
+        options = self.Parent._options
+        defaultFile = options['defaultFile'] if 'defaultFile' in options else ''
+        defaultPath = options['defaultPath'] if 'defaultPath' in options else ''
+        message = options['message'] if 'message' in options else ''
+        wildcard = options['wildcard'] if 'wildcard' in options else ''
+        return defaultFile, defaultPath ,message, wildcard

```

Comment:
I'd argue that you can get by without this method -- you're trading glorious open key/val maps for closed positional semantics. 

The former is open to new Gooey options in the future, the latter is coupled to the current length and order of the return value -- meaning, a change to `getChooserOptions` would require a change to every one of its call sites due to the tuple unpacking. 

So, let's just use the `options` map directly. Python gives an API for accessing a map with fallback defaults on key miss which'll get rid of the verbose ternary expressions and unpacking awkwardness. So, things should end up looking, more or less, like this:

```python 
    def getDialog(self):
        return wx.FileDialog(
            self,
            message=options.get('message', _('open_file')),
            style=wx.FD_OPEN | wx.FD_FILE_MUST_EXIST,
            wildcard=options.get('wildcard', wx.FileSelectorDefaultWildcardStr))
```